lib/Rex/CLI.pm

```diff
@@ -612,11 +612,20 @@ sub summarize {
   Rex::Logger::info( @failures . " out of " . @summary . " task(s) failed:",
     "error" );
 
-  Rex::Logger::info( "\t$_->{task} failed on $_->{server}", "error" )
-    foreach sort {
-         ncmp( $a->{task}, $b->{task} )
-      || ncmp( $a->{server}, $b->{server} )
-    } @failures;
+  foreach (
+    sort {
+           ncmp( $a->{task}, $b->{task} )
+        || ncmp( $a->{server}, $b->{server} )
+    } @failures
+    )
+  {
+    Rex::Logger::info( "\t$_->{task} failed on $_->{server}", "error" );
+    if ( $_->{error_message} ) {
+      for my $line ( split( $/, $_->{error_message} ) ) {
+        Rex::Logger::info( "\t\t$line", "error" );
```

Comment:
Interesting not sure how this looks.  Might be a good idea.  I think it depends on the how long he error message is.  I was thinking of it like prove where it reported on the # of errors and you scrolled up to see the errors in context.  I assumed the error message would be long and annoying to view inline.  I guess I'm a little dubious that this is going to be readable in all cases.